### ISSUE TITLE
FR: Additional Views

### ISSUE THREAD

**Issue by sstar0016** [mcndt/obsidian-toggl-integration#118]
Love this plugin, super clean and nice to work with, use it for daily reviews

Would be keen to get:
- Scalability of height (atm it takes up a lotta room, I would take smaller graphs for a bit more legroom)
-  different views (sorted by project, client, task etc) maybe like [timery](https://timeryapp.com/)?
![image](https://user-images.githubusercontent.com/124852454/218822844-095cfecc-88e6-49f1-8abf-cdd79c6d0ae2.png)
![image](https://user-images.githubusercontent.com/124852454/218822881-64995658-ecdc-46fc-8c67-fc4d7b62d0ec.png)

**Comment by mcndt** [comment]
I love timery! Could you give some concrete examples of visualisations that are missing that you would like to add?

**Comment by sstar0016** [comment]
A few features of timery I'd like to see in this plugin (listed in order of personal priority):

1. group list/graph by project/client/tags (instead of just as individual entries)
2. filter by project/client/tags (ie just graph the time entries for a certain client)
3. change list order of entries (sort by duration, date)

**Comment by mcndt** [comment]
For 1: I'll see if this is possible with Toggl's new summary API, but I would presume so. Ironically enough I recently also had the need to group summaries by tag rather than project for a work thing.

2: This is already possible 😄 
- [Documentation about project/client filtering](https://github.com/mcndt/obsidian-toggl-integration/wiki/Toggl-Query-Language-(TQL)-Reference#filtering-tags)
- [Documentation about tag filtering](https://github.com/mcndt/obsidian-toggl-integration/wiki/Toggl-Query-Language-(TQL)-Reference#filtering-tags)

3. This is also possible, see docs:
- [Documentation about sorting](https://github.com/mcndt/obsidian-toggl-integration/wiki/Toggl-Query-Language-(TQL)-Reference#filtering-tags)
- In summary views, the sorting orders by project time (e.g. "SORT DESC" orders the project from most to least time logged)
- In list views grouped by date, it sorts chronological according to the SORT keyword (e.g. SORT DESC sorts reverse-chronologically)
- In list views grouped by client/project, it sorts by time spent on that client/project.

**Comment by mcndt** [comment]
FYI there is also a quick reference to see everything that is possible: https://github.com/mcndt/obsidian-toggl-integration/wiki/TQL-Quick-Reference
